recipe/meta.yaml

```diff
@@ -72,14 +83,14 @@ requirements:
     - gast >=0.2.0
     - keras-applications >=1.0.6
     - keras-preprocessing >=1.0.5
-    - {{ pin_compatible('numpy') }}
+    - {{ pin_compatible('numpy') }}  # [linux]
```

Comment:
If numpy is behind a selector, the dependency on other platforms needs to be listed explicitly:
```
- numpy >=1.13.3   # [not linux]
```

Reply:
Thanks! Missed that one.